src/js/elements/point_tray/theme.js

```diff
@@ -29,12 +28,15 @@ export class Theme extends Component {
         $(this.element).find('.point-mapper__h1_checkbox input[type=checkbox]').on('click', () => self.toggle());
         $('.point-data__h1_name .truncate', this.element).text(this.name);
         $('.point-mapper__h1_trigger', this.element)
-            .removeClass(activeClsName)
-            .addClass('theme-' + this.themeIndex)
+            .removeClass('active')
+            .addClass('theme-' + this.themeIndex)   //remove this line
```

Comment:
Was this comment a todo?

Reply:
I knew I was forgetting something :) yes it was a todo and removed it. nice catch